packages/devextreme/js/localization/messages/es.json

```diff
@@ -195,6 +195,7 @@
         "dxDataGrid-masterDetail": "Cell with details",
 
         "dxTreeList-ariaTreeList": "Tree list with {0} rows and {1} columns",
+        "dxTreeList-ariaExpandableInstruction": "Use Ctrl + right arrow key combination to expand and Ctrl + left arrow key combination to collapse a focused node",
```

Comment:
```suggestion
        "dxTreeList-ariaExpandableInstruction": "Press Ctrl + right arrow key combination to expand the focused node and Ctrl + left arrow to collapse it",
```

Reply:
Thanks Vlada.. pushing commit now and waiting for CI check..